src/me/sirfaizdat/prison/mines/MinesManager.java

```diff
@@ -71,7 +74,34 @@ private void broadcastToWorld(String s, World w) {
     }
 
     public void load() {
-        for (File file : getAllMineFiles()) {
+    	boolean convert = Prison.i().config.useJson;
+    	if (!convert){Prison.l.warning("The new mine format converter is disabled");Prison.l.warning("Set the json option to true in the config.yml to enable it");}
```

Comment:
Part of me agrees, then again there will probably be somebody out there who will want to keep the old system. I say keep it (for now) for backwards compatibility.